### ISSUE TITLE
Accuracy of the model

### ISSUE THREAD

**Issue by Hariant** [hyuna915/squad#11]
Hi,
I have converted the code to run in latest python and tensorflow version. But not getting the same accuracy as mentioned the paper and as per the logs. Could you please share me the train data and glove embedding file ans if possible saved best model ?
